Resources/Prototypes/fonts.yml

```diff
@@ -45,3 +45,7 @@
 - type: font
   id: Emoji
   path: /Fonts/NotoEmoji.ttf
+
+- type: font
+  id: SignatureFont
+  path: /Fonts/Signature.ttf
```

Comment:
Where does this font come from? Also ideally this should be placed under a starlight specific font YML file along with the font file also being in a starlight folder